packages/e2e/cypress/e2e/createProjects.cy.ts

```diff
@@ -55,85 +57,99 @@ const warehouseConfig = {
+    cy.get('select[name="warehouse.http_scheme"]').select('https');
 
     // DBT
-    cy.get('[name="dbt.type"]').select('dbt local server');
-    cy.get('[name="dbt.target"]').type('test');
-    cy.get('[name="warehouse.schema"]').type(config.schema);
+    cy.get('select[name="dbt.type"]').select('dbt local server');
+    cy.get('input[name="dbt.target"]').type('test');
+    cy.get('input[name="warehouse.schema"]').type(config.schema);
 };
 
 const configureDatabricksWarehouse = (
     config: typeof warehouseConfig['databricks'],
 ) => {
-    cy.get('[name="warehouse.serverHostName"]').type(config.host, {
+    cy.get('input[name="warehouse.serverHostName"]').type(config.host, {
+        log: false,
+    });
+    cy.get('input[name="warehouse.httpPath"]').type(config.httpPath, {
         log: false,
     });
-    cy.get('[name="warehouse.httpPath"]').type(config.httpPath, { log: false });
-    cy.get('[name="warehouse.personalAccessToken"]').type(config.token, {
+    cy.get('input[name="warehouse.personalAccessToken"]').type(config.token, {
         log: false,
     });
 
     // DBT
-    cy.get('[name="dbt.type"]').select('dbt local server');
-    cy.get('[name="dbt.target"]').type('test');
-    cy.get('[name="warehouse.database"]').type(config.schema);
+    cy.get('select[name="dbt.type"]').select('dbt local server');
+    cy.get('input[name="dbt.target"]').type('test');
+    cy.get('input[name="warehouse.database"]').type(config.schema);
 };
 
 const configureSnowflakeWarehouse = (
     config: typeof warehouseConfig['snowflake'],
 ) => {
-    cy.get('[name="warehouse.account"]').type(config.account, { log: false });
-    cy.get('[name="warehouse.user"]').type(config.user, { log: false });
-    cy.get('[name="warehouse.password"]').type(config.password, { log: false });
-    cy.get('[name="warehouse.role"]').type(config.role);
-    cy.get('[name="warehouse.database"]').type(config.database);
-    cy.get('[name="warehouse.warehouse"]').type(config.warehouse);
+    cy.get('input[name="warehouse.account"]').type(config.account, {
+        log: false,
+    });
+    cy.get('input[name="warehouse.user"]').type(config.user, { log: false });
+    cy.get('input[name="warehouse.password"]').type(config.password, {
+        log: false,
+    });
+    cy.get('input[name="warehouse.role"]').type(config.role);
+    cy.get('input[name="warehouse.database"]').type(config.database);
+    cy.get('input[name="warehouse.warehouse"]').type(config.warehouse);
 
     // DBT
-    cy.get('[name="dbt.type"]').select('dbt local server');
-    cy.get('[name="dbt.target"]').type('test');
-    cy.get('[name="warehouse.schema"]').type(config.schema);
+    cy.get('select[name="dbt.type"]').select('dbt local server');
+    cy.get('input[name="dbt.target"]').type('test');
+    cy.get('input[name="warehouse.schema"]').type(config.schema);
 };
 
-const testCompile = () => {
+const testCompile = (): Cypress.Chainable<string> => {
```

Comment:
What's the advantage of using this chainable ?

Reply:
So we can chain other actions and pass the project uuid 

```
        testCompile().then((projectUuid) => {
            testFilterStringEscaping(projectUuid);
            testTimeIntervalsResults(projectUuid);
            testPercentile(projectUuid);
        });
```

Note that cypress actions are not Promises so I couldn't simply return the string.